### DISCUSSION TITLE
hi how show flyout menu

### DISCUSSION THREAD

**Discussion by luca233** [RadekVyM/SimpleToolkit#17] [General]
hi friends i need implemente simple shell , but i not find how show flyout menu

please can you tell me what i need do for get menu flyout

thanks

Ulises

**Comment by RadekVyM** [comment]
Hi @luca233,

apologies for the delayed response. I have recently built a sample - [Hamburger Menu App](https://github.com/RadekVyM/HamburgerMenuApp) - demonstrating how to implement custom flyout menu using `SimpleShell`.

**Reply by Usandria** [reply]
thanks
